tools/utils.go

```diff
@@ -132,27 +132,44 @@ func GetMainTemplate(
 	key string,
 	sc string,
 	sleepTime uint,
+	export bool,
```

Comment:
boolean should have a verb inside the variable name

in this case, i think `shouldExport` is good